### PR TITLE
Hide labels for blobs

### LINKED ISSUE #179: Hide labels for blobs

### LINKED ISSUE #196: Improve tooltip performance

### LINKED ISSUE #197: Extract drawChart away from DrawChart body

### PR DESCRIPTION
**Pull request by joglr**
Fixes #179
Fixes #196
Fixes #197

### CODE REVIEW

**Review by tjomson — CHANGES_REQUESTED** [review]
Clicking a node causes this error
billede

**Comment by emiljapelt** [comment]
Filter function

.filter((d) => d.data.type === "tree" || (d.data as HydratedGitBlobObject).noLines >= 40)

**Comment by hojelse** [comment]
Legend component doesn't show up

legend_is_gone

**Review by hojelse — CHANGES_REQUESTED** [review]
Legend component doesn't show up

legend_is_gone

File: app/src/components/Tooltip.tsx [hunk 1]
```diff
@@ -11,28 +11,35 @@ const TooltipBox = styled(Box)`
...
   will-change: transform visibility;
+  pointer-events: none;
 `
 
-export function Tooltip() {
-  const { currentHoveredBlob } = useStore()
+const TooltipContainer = styled.div`
+  position: absolute;
+  inset: 0;
+  pointer-events: none;
```
Comment (joglr) [inline app/src/components/Tooltip.tsx:20]: This still overflows the body

**Review by hojelse — APPROVED** [review]

**Review by tjomson — APPROVED** [review]
🚀

**Pull request merged**